Druid/CCodeGenerator.extension.st

```diff
@@ -0,0 +1,16 @@
+Extension { #name : #CCodeGenerator }
+
+{ #category : #'*Druid' }
+CCodeGenerator >> inferTypes [
+	
```

Comment:
This method should not be moved to this package!

Reply:
Fixed

Reply:
Ok done